lib/cuckoo/common/office/rtfdump.py

```diff
@@ -47,17 +47,21 @@
 import zipfile
 import binascii
 import textwrap
-import re
 import string
 import hashlib
 import json
 from io import StringIO
 import collections
 import glob
 
+try:
+    import re2 as re
+except ImportError:
+    import re
+
 try:
     import yara
-except Exception:
+except ImportError:
```

Comment:
@doomedraven I think you're referring to here where I replaced `except Exception` with `except ImportError`

Reply:
em ok than the cherry-picking should did a mess, thanks